distribution/kernel/carbon-home/repository/resources/conf/templates/repository/conf/carbon.xml.j2

```diff
@@ -821,4 +821,19 @@
         <disableRestart>{{server.disable_restart_from_ui | default('false')}}</disableRestart>
      </disableShutdownRestartFromUI>
 
+     {% if correlation_logs is defined %}
+     <!-- Configure correlation logs -->
+     <CorrelationLogs>
+         <!-- Enable/disable correlation logs -->
+         <enable>{{correlation_logs.enable}}</enable>
+         <!-- Comma seperated list of components which correlation logs need to be enabled. Possible values are
+         jdbc, http, gw, apimgt, keymgt and tokenmgt -->
+         <components>{{correlation_logs.components}}</components>
+         <!-- Comma seperated list of threads, which correlation logging should ignore. This applies only for HTTP.  -->
+         <blacklistedThreads>{{correlation_logs.blacklistedThreads}}</blacklistedThreads>
```

Comment:
Shall we use "deniedThreads" instead of blacklisted?
```suggestion
         <blacklistedThreads>{{correlation_logs.blacklistedThreads}}</blacklistedThreads>
```

Reply:
Fixed